docker/runtime/readme.md

```diff
@@ -0,0 +1,48 @@
+# Running Azure IoT Edge in a container
+
+## Notes
+
+The following procedure has been tested on Windows 10 machine only. To avoid unexpected application crashes, run everything from Powershell prompts.
```

Comment:
Didn't someone add support for Mac / Linux?

Reply:
Yep, removed the "Note" section, it's not needed anymore